CHANGELOG.rst

```diff
@@ -14,8 +14,13 @@ Change Log
 Unreleased
 ----------
 
+[0.34.1] - 2017-06-06
```

Comment:
Let's coordinate the version bump here with https://github.com/edx/edx-enterprise/pull/123.